docs/analytics/auto_scroll_tracker.md

```diff
@@ -0,0 +1,52 @@
+# Scroll tracking
+
+The analytics code includes the ability to add scroll tracking to specified pages for users who have consented to cookies.
+
+There are currently two scroll tracking scripts in the gem. The new one has been written to address problems with the old one, but the scroll tracking handled by the old one has yet to be transferred to the new one.
```

Comment:
Rather than calling it "the new one" which seems to often age badly - it might be worth saying.

There are currently two scroll tracking scripts in the gem. You should use AutoScrollTracker. The previous one, ScrollTracker, has been deprecated by the introduction of AutoScrollTracker and should now be considered legacy.

The new one has been written to address problems with the old one, but the scroll tracking handled by the old one has yet to be transferred to the new one

Reply:
Updated, thanks.